src/settings/SettingsAddress.js

```diff
@@ -58,18 +57,7 @@ export default class SettingsAddress extends React.Component {
 			address_postcode: state.postcode,
 			phone: state.tel,
 		};
-		LiveData.api.setAccountSettings(req).then(
-			response => {
-				if (response.set_settings === 1) {
-					this.actions.updateSettingFields(req);
-				} else {
-					SettingsAddress.handleUpdateError(response);
-				}
-			},
-			response => {
-				SettingsAddress.handleUpdateError(response);
-			}
-		);
+		this.props.actions.attemptUpdateSettings(req, SettingsAddress.handleUpdateError);
```

Comment:
tryUpdateSettings is more canonical.
I would still suggest it is updateSettings, as usually try/attempt indicates an alternative call exists without it, one throwing, another not.
After all, most calls might fail for some reason, lets not call them attempt?